src/main/java/org/dbflute/intro/app/logic/exception/DirectoryDoesNotExsistException.java

```diff
@@ -0,0 +1,10 @@
+package org.dbflute.intro.app.logic.exception;
+
+public class DirectoryDoesNotExsistException extends Exception {
```

Comment:
I want you to completely unify the wording.
"Not Found" or "Does not exists.".

Reply:
Unified to "Not Found".

Exception classes created↓↓
src/main/java/org/dbflute/intro/app/logic/exception/DirNotFoundException.java
src/main/java/org/dbflute/intro/bizfw/tellfailure/OpenDirNotFoundException.java

Methods of IntroMessages.java↓↓
https://github.com/dbflute/dbflute-intro/pull/250/files#diff-117609f640468afd4df5b64e73cc55c7891c748c34e062ac584b2e6151fae957R898